packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py

```diff
@@ -563,36 +608,194 @@ def clear_job_annotations(job_id: int) -> None:
             raise
 
 
-def update_job_assignee(id: str, assignee_id: int | None):
+def setup_gt_job(task_id: int, dataset_path: Path, format_name: str) -> None:
+    gt_job = get_gt_job(task_id)
+    upload_gt_annotations(gt_job.id, dataset_path, format_name=format_name)
+    finish_gt_job(gt_job.id)
+    settings = get_quality_control_settings(task_id)
+    update_quality_control_settings(settings.id)
```

Comment:
Consider moving the function into `job_creation.py`. This file is supposed to contain generic functions without use case logic.

Reply:
@zhiltsov-max, Could you please mark comments that you've fixed yourself in your PR?  Otherwise, it's not obvious for me which comments I should fix.